src/appkit.cs

```diff
@@ -1565,6 +1602,10 @@ partial interface NSBitmapImageRep : NSSecureCoding {
 		[Field ("NSImageEXIFData")]
 		NSString EXIFData { get; }
 
+		[Mac (12,0)]
+		[Field ("NSImageIPTCData")]
+		NSString IPTCData { get; }
```

Comment:
[IPTC](https://en.wikipedia.org/wiki/IPTC_Information_Interchange_Model) is an acronym that's longer that 2 letters, so:

```suggestion
		NSString IptcData { get; }
```